cmds/kernelimg.c

```diff
@@ -0,0 +1,99 @@
+/*
+ * Phoenix-RTOS
+ *
+ * Operating system loader
+ *
+ * Load kernel binary image
+ *
+ * Copyright 2021 Phoenix Systems
+ * Author: Aleksander Kaminski
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+#include "cmd.h"
+#include "elf.h"
+
+#include <hal/hal.h>
+#include <lib/lib.h>
+#include <phfs/phfs.h>
+#include <syspage.h>
+
+
+static void cmd_kernelimgInfo(void)
+{
+	lib_printf("loads Phoenix-RTOS binary image, usage: kernelimg [<dev> [name] <text begin> <text size> <data begin> <data size>]");
```

Comment:
```suggestion
	lib_printf("loads Phoenix-RTOS binary image, usage: kernelimg <dev> [name] <text begin> <text size> <data begin> <data size>");
```
Calling only with 5 or 6 arguments.

Reply:
Done